res/mock_eth_connector/src/admin_controlled.rs

```diff
@@ -0,0 +1,70 @@
+#![allow(dead_code)]
+use near_sdk::AccountId;
+
+pub type PausedMask = u8;
+
+/// Admin control flow flag indicates that all control flow unpause (unblocked).
+pub const UNPAUSE_ALL: PausedMask = 0;
+/// Admin control flow flag indicates that the deposit is paused.
+pub const PAUSE_DEPOSIT: PausedMask = 1 << 0;
+/// Admin control flow flag indicates that withdrawal is paused.
+pub const PAUSE_WITHDRAW: PausedMask = 1 << 1;
+
+pub trait AdminControlled {
```

Comment:
I think this trait was present in the engine code as well. Is it replicated here so that the engine library is not pulled in?

Reply:
It is not Engine. It's `aurora-eth-conenctor` contract. And it can't be pulled,